app/marketing/mails.py

```diff
@@ -1175,7 +1173,17 @@ def new_bounty_daily(bounties, old_bounties, to_emails=None):
         try:
             setup_lang(to_email)
             from_email = settings.CONTACT_EMAIL
-            html, text = render_new_bounty(to_email, bounties, old_bounties, trending_quests=trending_quests())
+
+            from marketing.views import quest_of_the_day, upcoming_grant, upcoming_hackathon, latest_activities
```

Comment:
Move the imports out of the function itself for performance unless needed to prevent a circular import

Reply:
Moving the imports out of the function gives the error:
```
  File "/code/app/dashboard/models.py", line 66, in <module>
    from marketing.mails import featured_funded_bounty, fund_request_email, start_work_approved
  File "/code/app/marketing/mails.py", line 50, in <module>
    from marketing.views import quest_of_the_day, upcoming_grant, upcoming_hackathon, latest_activities
  File "/code/app/marketing/views.py", line 44, in <module>
    from chat.tasks import update_chat_notifications
  File "/code/app/chat/tasks.py", line 8, in <module>
    from dashboard.models import Bounty, HackathonEvent, HackathonRegistration, HackathonSponsor, Profile
ImportError: cannot import name 'Bounty'
```